source/pdo_sqlsrv/pdo_util.cpp

```diff
@@ -421,6 +421,10 @@ pdo_error PDO_ERRORS[] = {
         SQLSRV_ERROR_OUTPUT_PARAM_TYPES_NOT_SUPPORTED,
         { IMSSP, (SQLCHAR*) "Stored Procedures do not support text, ntext or image as OUTPUT parameters.", -83, false }
     },
+    {
+        SQLSRV_ERROR_ENCRYPTED_STREAM_FETCH,
+        { IMSSP, (SQLCHAR*) "Connection with Column Encryption enabled does not support fetching stream. Please fetch the data as a string.", -84, false }
+    },
     { UINT_MAX, {} }
```

Comment:
Do we have to add this error in PDO when it doesn't support returning streams?

Reply:
Since the error is thrown in the core code, it needs to be here to compile.